config/bidderinfo.go

```diff
@@ -206,6 +209,11 @@ type InfoReaderFromDisk struct {
 	Path string
 }
 
+const (
+	ResponseFormatIFrame   = "b" // b = blank HTML response
+	ResponseFormatRedirect = "i" // i = image response
+)
```

Comment:
Could these be defined in the usersync package and referenced here, or does that introduce a circular reference?

Reply:
Yeah I tried to, but it introduced a circular reference